test/bdd/steps/lib/otnode.js

```diff
@@ -58,6 +58,8 @@ class OtNode extends EventEmitter {
         this.state.holdingData = [];
         // Verified escrows for own imports. List of import IDs.
         this.state.escrowsVerified = [];
+        // grab node's wallet address
+        this.state.nodesWalletAddress = null;
```

Comment:
FYI you can use node's wallet from configuration since it was given in advance.

Reply:
This one can be check against given configuration.

Reply:
ok, acknowledged for future.